.github/workflows/basic.yml

```diff
@@ -148,8 +150,6 @@ jobs:
     name: Windows
     env:
       VCPKG_PATH: "${{github.workspace}}/build/vcpkg/packages/hwloc_x64-windows;${{github.workspace}}/build/vcpkg/packages/tbb_x64-windows;${{github.workspace}}/build/vcpkg/packages/jemalloc_x64-windows"
```

Comment:
we could use ${{env.BUILD_DIR}} here

Reply:
We cannot, it would be a syntax error, because GHA does not allow to use env vars inside a definition of another env var.